quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+```shell
+mysql --version
+```
 
-</div>
-<div label="Linux">
+You should see an output indicating the MySQL client version number.
 
-As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB instances, 3 TiKV instances, 3 PD instances, and optional TiFlash instances. With TiUP Playground, you can quickly build the test cluster by following these steps:
+If you already have the MySQL client installed, you can connect to TiDB with:
 
-1. Download and install TiUP:
 
-    {{< copyable "shell-regular" >}}
+```shell
+mysql --host 127.0.0.1 --port 4000 -u root
+```
 
-    ```shell
-    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
-    ```
+## Understanding what just happened
 
-    If the following message is displayed, you have successfully installed TiUP:
-
-    ```log
-    Successfully set mirror to https://tiup-mirrors.pingcap.com
-    Detected shell: zsh
-    Shell profile:  /Users/user/.zshrc
-    /Users/user/.zshrc has been modified to add tiup to PATH
-    open a new terminal or source /Users/user/.zshrc to use it
-    Installed path: /Users/user/.tiup/bin/tiup
-    ===============================================
-    Have a try:     tiup playground
-    ===============================================
-    ```
+You just started to use a fully functional distributed database cluster with:
 
-    Note the Shell profile path in the output above. You need to use the path in the next step.
+- **TiDB Server:** SQL layer that processes queries and speaks MySQL protocol
+- **TiKV:** Distributed storage layer that replicates data automatically and handles transactions across nodes
+- **PD (Placement Driver):** Cluster orchestrator that works behind the scenes to coordinate TiDB, TiKV, and TiFlash components, managing cluster metadata and ensuring data is distributed optimally
 
-2. Declare the global environment variable:
+Additionally, `tiup playground` installed optional components:
 
-    > **Note:**
-    >
-    > After the installation, TiUP displays the absolute path of the corresponding Shell profile file. You need to modify `${your_shell_profile}` in the following `source` command according to the path.
+- **TiFlash:** Real-time analytics engine that accelerates complex queries and reporting without slowing down your live application
+- Built-in **monitoring stack** with:
+  - TiDB Dashboard (default: http://127.0.0.1:2379/dashboard, user: `root`, no password)
+  - Grafana + Prometheus (default: http://127.0.0.1:3000, user: `admin`, password: `admin`)
 
-    {{< copyable "shell-regular" >}}
+You can verify your running database cluster components with:
 
-    ```shell
-    source ${your_shell_profile}
-    ```
+```shell
+tiup playground display
```

Comment:
Could we instruct users to run this command before introducing the components? Then, users will see the following output in their terminal first and see the explanations of these components in our document. 

```
A new version of playground is available: v1.14.1 -> v1.16.1

    To update this component:   tiup update playground
    To update all components:   tiup update --all

Pid    Role     Uptime
---    ----     ------
18997  pd       50m54.278572s
19089  tikv     50m6.9280185s
19353  tidb     47m42.160680084s
20593  tiflash  36m26.515627792s
```